.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,20 @@
+---
+name: Feature request
+about: Suggest an idea or enhancement to the ohsome API.
+title: ''
+labels: 'feature request'
+assignees: ''
+
+---
+
+### Problem Description
+Please add a clear and concise description of what the problem is, e.g. "I'm always frustrated when..., because..."
```

Comment:
I don't like the negative connotation in this section too much. Most of the time feature requests are not about problems (that's what a bug report is for), but instead focus on potential improvements. 

For feature request, users often forget to state their intention/rationale behind suggested ideas. This makes it often hard to evaluate feature requests objectively. I would prompt the user to try to formulate their suggestion as a user story, perhaps like this:

```suggestion
### Feature Description
Please add a clear and concise description of what your idea is, what use cases it solves. You could try to formulate it as a user story using the following template: _“As a … I would like to … so that …”_ (e.g., _“As a researcher I would like to request the daily number of OSM changesets touching particular map features in a given area in order to analyse trends in mapping activity.”_).
```

Reply:
I don't like to put user story models here. I think the mentioned example protracts too much the description and it isn't very useful for the users, since most of the features requested here are small and specific enhancements.

Reply:
ok, user-story style might be overkill, fair enough. But my point was that the current wording isn't about _enhancements_ at all. It reads more like a bug report, talking about _problems_ and _frustration_.

Can you come up with a better wording which is tailored towards _small and specific enhancements_?